### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by jgrewe**
This PR contains
* some changes to the README and the index.md used for the github.io pages
* removal of unused md files
* a test to include doxygen api doc to read the docs/sphinx

### CODE REVIEW

File: docs/code/spike_times.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    std::mt19937 gen(rd()); //Standard mersenne_twister_engine seeded with rd()
+    std::uniform_real_distribution<> dis(0.0, 1.0);
+    for (int n = 0; n < 10; ++n) {
+        // Use dis to transform the random unsigned int generated by gen into a
+        // double in [1, 2). Each call to dis(gen) generates a new random double
+        std::cout << dis(gen) << ' ';
+    }
+    std::cout << '\n';
+    */
+}
```
Comment (gicmo) [inline docs/code/spike_times.cpp]: Uncommented on purpose?

File: docs/code/spike_times.cpp [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+void test_times(size_t runs, double duration, double dt, size_t trials=10, size_t chunksize=1000) {
+    for (size_t i = 0;i < runs; ++i) {
+        //std::string filename = "Spike_time_benchmark_times_" + nix::util::numToStr(i) + ".nix";
+        nix::File f_times = nix::File::open("test.nix", nix::FileMode::Overwrite);//, "hdf5",
+        //                                            nix::Compression::DeflateNormal);
+        nix::Block b_times = f_times.createBlock("Test", "test");
+        //nix::NDSize initial_shape(1, chunksize);
+        // nix::DataArray da_times = b_times.createDataArray("spike times", "nix.event.spike_times",
+        //nix::DataType::Double, initial_shape);
```
Comment (gicmo) [inline docs/code/spike_times.cpp]: Uncommented?

File: README.md [hunk 3]
```diff
@@ -21,6 +21,20 @@ information](https://github.com/G-Node/nix/wiki)
 
 The current implementations store the actual data using the [HDF5](http://www.hdfgroup.org/) file format as a storage backend.
 
+Documentation --- where to find which information?
+--------------------------------------------------
+
+Project documentation is split up into three parts:
+1. Technical information: Please visit the GitHub repository [Wiki](https://github.com/G-Node/nix/wiki).
+2. General Introduction and tutorial: Visit the project pages on [ReadTheDocs](https://nixio.readthedocs.io/en/latest/)
+3. API documentation.
```
Comment (gicmo) [inline README.md]: Where is that? ;)

Comment (achilleas-k) [reply README.md]: I think the API docs aren't available right now, are they?  Figured we can add the link when we're done sorting.  Maybe add a note (or issue) about it.

File: docs/code/spike_times.cpp (hunk 1 above)
Comment (jgrewe) [reply docs/code/spike_times.cpp]: not yet done, will be used in the future

File: README.md (hunk 3 above)
Comment (jgrewe) [reply README.md]: yes, I want to see, if the doxygen sphinx interaction works on read the docs. When it does the link will be set

Comment (jgrewe) [reply README.md]: at the moment the API doc is actually nowhere

File: docs/code/spike_times.cpp (hunk 2 above)
Comment (jgrewe) [reply docs/code/spike_times.cpp]: see above, this is not yet done. Can remove it from the PR, if yoou want, otoh, it does not hurt in the docs/code, imho

Comment (jgrewe) [reply docs/code/spike_times.cpp]: see above, this is not yet done. Can remove it from the PR, if you want, otoh, it does not hurt in the docs/code, imho

**Pull request merged**